src/typings/editContext.d.ts

```diff
@@ -5,8 +5,7 @@
 
 type DOMString = string;
 
-declare class EditContext extends EventTarget {
-	constructor(options?: EditContextInit);
+interface EditContext extends EventTarget {
```

Comment:
Curious why we still have the `editContext.d.ts` here in the typings folder?

Reply:
Yes so the compiler was saying that:

_editContext is not defined on HTML elements_

so then I figured it must be because I needed a d.ts file with the following:

interface HTMLElement {
   editContext?: <type>
}

which would be merged with the HTMLElement interface

Now I wanted to avoid defining the editContext to be of type _any_, as I wanted it to be of the specific type of the EditContext so I added the interface for EditContext before it.

Please let me know if I am misunderstanding something.